### PR TITLE
fix #166: use hidpi images for hidpi screens

### LINKED ISSUE #166: Blurry icon in location bar (HiDPI issue)
The shield icon is very blurry on my MacBook with Retina display. A icon with a higher resolution should be used for HiDPI displays.

### PR DESCRIPTION
**Pull request by groovecoder**
@pdehaan or @rbillings - who wants to spot-check this on a HiDPI screen now?

### CODE REVIEW

**Comment by pdehaan** [comment]
Not sure if these will work well after screenshotting, but here's what I'm currently seeing with my special eyes:

download_firefox_ _free_web_browser_ _mozilla
**Figure 1:** 27" non-retina Dell.

download_firefox_ _free_web_browser_ _mozilla_and_fix__166__use_hidpi_images_for_hidpi_screens_by_groovecoder_ _pull_request__203_ _mozilla_blok
**Figure 2:** 15" rMBP.

**Comment by pdehaan** [comment]
TLDR: 👍 Looks great on my rMBP.

**Comment by groovecoder** [comment]
Thanks @fangshih for the HiDPI assets!
